iOS/README.md

```diff
@@ -1,5 +1,7 @@
 # DuckDuckGo iOS
 
+Temp change to force iOS workflows to run
+
```

Comment:
Will delete this before merging, assuming this fix works. Need to update the workflows to also run on a modified `.github` directory, rather than having to make temp changes like this.